Packs/Whois/Integrations/Whois/Whois.yml

```diff
@@ -57,14 +58,6 @@ configuration:
   section: Collect
   advanced: true
   required: false
-- display: Suppress Rate Limit errors
```

Comment:
We're no longer parsing the suppress integration instance in our Python code.

Is this OK to remove the integration setting? It's not a required field.

Reply:
@tkatzir can you have a look?

Reply:
AFAIK, the "standard" to remove a config is to make it hidden.

Reply:
After discussing this with @ShahafBenYakir, it's ok.
My main concern was about breaking custom code.

Reply:
> removing this field yields another break in BC for customers no? Customers that had this flag on will now get errors instead warnings no?

The usage of the flag in version 1.4.16 is in this section of the `get_whois_ip`:

```python
try:
        return ip_obj.lookup_rdap(depth=1, retry_count=retry_count, rate_limit_timeout=rate_limit_timeout)
except urllib.error.HTTPError as e:
        if rate_limit_errors_suppressed:
            demisto.debug(f'Suppressed HTTPError when trying to lookup rdap info. Error: {e}')
            return
        demisto.error(f'HTTPError when trying to lookup rdap info. Error: {e}')
        raise e
```

So basically if the flag is on and we get some type of generic HTTP error, we return `None` from the `get_whois_ip` command. However, we never actually check the value of `response` in `ip_command` method:

```python
        response = get_whois_ip(args)

        result = CommandResults(
            outputs_prefix='Whois.IP',
            outputs_key_field='query',
            outputs=response,
            readable_output=tableToMarkdown('Whois results:', readable_data),
            raw_response=response,
            indicator=ip_output
        )

        results.append(result)
    return results
```

And in `main` we check if the `ret_value` exists (which it always does):
```python
        if command == 'ip':
            demisto_args = demisto.args()
            ip = demisto_args.get('ip')
            ret_value = ip_command(ip, reliability)
            if ret_value:
                return_results(ret_value)
            else:
                return_error(f'Failed to lookup ip {ip}')
```

So an error is never returned in that case, just an empty command result.

 If the suppress flag is off and an exception occurs, we raise the exception and it gets propagated and handled in the `main` `try/catch` block where a `return_error` is specified.

With the changes in this PR (including removal of the suppress flag) we now return an error entry when the `should_error` parameter is set and in cases when:

* The response is empty from `get_whois_ip` method.
* An exception (including `HttpError`) is raised 

If the `should_error` parameter is unset, we return a warning entry in those cases.

Reply:
Had a short call with @ShahafBenYakir about this. We decided to bring back the suppress flag so now if an `urrlib.error.HttpError` is encountered, we will return `None` from the `get_whois_ip` method and use the `with_error` integration instance to decide whether to return an error or warning entry. 

Changes done in https://github.com/demisto/content/pull/27370/commits/1176153aa910e031e45afd0410f348352d7a563e.